charts/clamav/templates/clamav-deployment.yaml

```diff
@@ -0,0 +1,129 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "clamav.fullname" . }}
+  labels:
+    {{- include "clamav.labels" . | nindent 4 }}
+spec:
+  revisionHistoryLimit: 1
+  replicas: {{ .Values.clamav.scaling.minReplicas }}
+  selector:
+    matchLabels:
+      name: {{ include "clamav.fullname" . }}
+  template:
+    metadata:
+      labels:
+        name: {{ include "clamav.fullname" . }}
+    spec:
+      containers:
+      - name: clamav
+        image: "{{ .Values.clamav.image }}:{{ default .Chart.AppVersion .Values.clamav.version }}"
+        command: ['clamd']
+        ports:
+        - containerPort: 3310
+          name: api
+          protocol: TCP
+        resources: {{ toYaml .Values.clamav.resources | nindent 10 }}
+        volumeMounts:
+          - mountPath: /var/lib/clamav
+            name: signatures
+          - mountPath: /etc/clamav
+            name: config-volume
+      - name: clamav-http
+        image: "{{ .Values.clamavHTTP.image }}:{{ default .Chart.AppVersion .Values.clamavHTTP.version }}"
+        ports:
+        - containerPort: 8080
+          name: api
+          protocol: TCP
+        livenessProbe:
+          httpGet:
+            path: /v1alpha/healthz
+            port: 8080
+          initialDelaySeconds: 10
+          timeoutSeconds: 5
+          periodSeconds: 5
+          failureThreshold: 10
+        readinessProbe:
+          httpGet:
+            path: /v1alpha/healthz
+            port: 8080
+          initialDelaySeconds: 10
+          timeoutSeconds: 5
+          periodSeconds: 5
+          failureThreshold: 10
+        resources: {{ toYaml .Values.clamav.resources | nindent 10 }}
```

Comment:
Maybe this should have seperate resources than the actual clamav container?

Reply:
Good spot, changed